ai-ml/ray/terraform/examples/pytorch/main.tf

```diff
@@ -0,0 +1,122 @@
+provider "aws" {
+  region = local.region
+}
+
+provider "kubernetes" {
```

Comment:
This approach comes with pre-requisites and I don't see this approach solved the original problem. Data source is working fine and no issues so far. 

```tf
provider "kubernetes" {
  host                   = module.eks_blueprints.eks_cluster_endpoint
  cluster_ca_certificate = base64decode(module.eks_blueprints.eks_cluster_certificate_authority_data)
  token                  = data.aws_eks_cluster_auth.this.token
}
```